alphagov/paas-cf
# WIP: [#117152485] rename "stage" environment to "staging"

## What

There's currently a discrepancy in what we call the staging environment. The AWS account is called "stage", but the deployment (and domains etc) are called "staging".

We want everything to use staging

## How to review and test

This PR cannot be "tested". You must check that the code make sense, and then apply the procedure below to apply the changes. 

**Note**: You must be familiar with the deployments and have AWS full permissions.

## How to apply the changes

You must follow the procedure described in the [description of the card](https://www.pivotaltracker.com/story/show/117152485)

## Who can review

**Note**: You must be familiar with the deployments and have AWS full permissions.

Anyone but @keymon or @alext
